src/routes/game/GameView.vue

```diff
@@ -1267,8 +1208,6 @@ export default {
     position: relative;
     background-color: rgba(255, 255, 255, 0);
     &.reveal-top-two {
-      height: auto;
-      align-self: start;
```

Comment:
These were causing theming to jump around when we switched back and forth to revealing/hiding the top two. 

TODO: investigate how this looks in Firefox and safari because I think it may have been added to fix FF not applying height to the cards